schema.graphql

```diff
@@ -89,6 +89,7 @@ type Reward @entity {
   id: ID!
 	asset: Bytes!
 	committed: BigInt!
+  token: Token!
```

Comment:
this isn't a required field -- so can remove the `!`

Reply:
Okay